strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/configuration/StoragesConfigurationController.java

```diff
@@ -93,22 +99,25 @@ public StoragesConfigurationController(ConfigurationManagementService configurat
         this.repositoryIndexManager = repositoryIndexManager;
     }
 
-    @ApiOperation(value = "Add/update a storage.")
-    @ApiResponses(value = { @ApiResponse(code = 200, message = "The storage was updated successfully."),
+    @ApiOperation(value = "Adds a storage.")
+    @ApiResponses(value = { @ApiResponse(code = 200, message = "The storage was created successfully."),
                             @ApiResponse(code = 500, message = "An error occurred.") })
     @PreAuthorize("hasAuthority('CONFIGURATION_ADD_UPDATE_STORAGE')")
-    @PutMapping(consumes = MediaType.APPLICATION_JSON_VALUE, produces = { MediaType.TEXT_PLAIN_VALUE,
-                                                                          MediaType.APPLICATION_JSON_VALUE })
-    public ResponseEntity saveStorage(
+    @PutMapping(consumes = MediaType.APPLICATION_JSON_VALUE,
+                produces = { MediaType.TEXT_PLAIN_VALUE,
+                             MediaType.APPLICATION_JSON_VALUE })
+    public ResponseEntity createStorage(
```

Comment:
Cosmetic: Could you please re-indent the parameters?

Reply:
How? Aren't they already indented?

Reply:
The parameters are starting on a new line the way you have it.
It's just code formatting... otherwise the pull looks good, thank you!